language/llama2-70b/README.md

```diff
@@ -63,32 +63,8 @@ Inside the container, set up the environment with `bash build.sh`. This will ins
 CPU-only setup, as well as any GPU versions for applicable libraries like PyTorch.
 
 
-## Get Model
-+ For now, MLCommons is not hosting the checkpoint, so you must first go to [llama2-request-link](https://ai.meta.com/resources/models-and-libraries/llama-downloads/) and make a request, sign in to huggingface (if you don't have account, you'd need to create one). **Please note your authentication credentials** as you may be required to provide them when cloninng below
-+ Requires Git Large Files Storage
-```
-export CHECKPOINT_PATH=${PWD}/Llama-2-70b-chat-hf
-git lfs install
-git clone https://huggingface.co/meta-llama/Llama-2-70b-chat-hf ${CHECKPOINT_PATH}
-
-```
-
-## Get Dataset
-
-```
```

Comment:
I think that procedure we have followed to generate the dataset can stay in README.

My ask is to provide rclone/wget command for downloading the dataset (it's not under the licence from Meta so we can distribute) and provide its md5sum. See https://github.com/mlcommons/inference/issues/1636

Reply:
@szutenberg The goal of this PR is to provide fix the broken link issue. I'll open another PR for downloading the processed dataset.

@nathanw-mlc can you leave the way to obtain the raw dataset and process it there

Reply:
Added back the old instructions and added new instructions for downloading the preprocessed dataset.